src/layout/verticalPanel/searchResultList.ts

```diff
@@ -621,10 +459,12 @@ export class SearchResultList extends St.BoxLayout {
 
         this.results = {};
         this.entrySelected = null;
-        this.remove_all_children();
+        for (const providerDisplay of this.providerDisplayMap.values()) {
+            providerDisplay.updateSearch([], []);
+        }
+        //this.remove_all_children();
```

Comment:
Remove debug code?

Reply:
This is a reset code not debug but I can add a comment and remove the commented code